### ISSUE TITLE
Upgrade to React 0.14.X

### ISSUE THREAD

**Issue by demigor** [reapp/reapp-routes#11]
Reapp-routes is the only component of reapp, which breaks dependencies on new react (0.14.X), rendering reapp unusable.

70684 error peerinvalid The package react@0.14.3 does not satisfy its siblings' peerDependencies requirements!
70684 error peerinvalid Peer omniscient@3.3.0 wants react@>=0.12.0 || >=0.14.0-alpha
70684 error peerinvalid Peer react-router@0.13.5 wants react@0.13.x||0.14.x
70684 error peerinvalid Peer reapp-routes@0.10.12 wants react@0.13.x
70684 error peerinvalid Peer reapp-ui@0.12.68 wants react@0.14.x

**Comment by ohadisr** [comment]
Please help on supporting react 0.14.x

**Comment by yms9654** [comment]
:+1:

**Comment by aprihodko** [comment]
Yep. Still cannot install dependencies as they require different react versions

**Comment by yemel** [comment]
:+1:
